ts/src/coinex.ts

```diff
@@ -4573,15 +4663,24 @@ export default class coinex extends Exchange {
         }
         if (code !== undefined) {
             currency = this.safeCurrencyCode (code);
-            request['asset'] = currency['id'];
+            request['asset'] = currency;
```

Comment:
@sc0Vu safeCurrencyCode returns the unified code, so don't think it is the most appropriate method here